src/lib/viewers/BaseViewer.js

```diff
@@ -1095,12 +1099,13 @@ class BaseViewer extends EventEmitter {
         this.annotationControls.setMode(nextMode);
 
         if (this.options.enableAnnotationsDiscoverability) {
-            switch (mode) {
+            switch (nextMode) {
+                case AnnotationMode.NONE:
```

Comment:
Should we add `NONE` case? We still want to select texts in `NONE` (default) mode, right?

Reply:
Yep, fixed

Reply:
I don't think we want to apply this class for the `NONE` case because this class turns off pointer-events for the textLayer span's which is what we want only when in explicit region mode

Reply:
I actually mean "should we remove `NONE` case?"...